checkov/terraform/module_loading/loaders/versions_parser.py

```diff
@@ -2,7 +2,7 @@
 from typing import List
 
 from packaging import version
-VERSION_REGEX = re.compile(r'^(?P<operator>=|!=|>=|>|<=|<|~>)*(?P<version>.+)$')
+VERSION_REGEX = re.compile(r'^(?P<operator>=|!=|>=|>|<=|<|~>)?(?P<version>.+)$')
```

Comment:
@gruebel Trying to improve this even further - how about this?
```suggestion
VERSION_REGEX = re.compile(r'^(?P<operator>=|!=|>=|>|<=|<|~>)?\s*(?P<version>[\d\.]+)$')
```